### PR TITLE
Logfiles

### PR DESCRIPTION
**Pull request by SamYoules**
These changes reduce the number of "computing xi" percentage-complete statements printed to the log file by a factor of 1000, to make the text files smaller.

### CODE REVIEW

File: py/picca/utils.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def smooth_cov(xi,
     sum_correlation = {}
     counts_correlation = {}
     for index in range(num_bins):
-        userprint("\rsmoothing {}".format(index), end="")
+        userprint("\rsmoothing {}".format(index) )
```
Comment (julianbautista) [inline py/picca/utils.py:110]: I think we can keep the `, end="")` in all instances in this file.

File: bin/picca_metal_xdmat.py [hunk 2]
```diff
@@ -287,12 +287,12 @@ def main():
         r_comov_min = cosmo.get_r_comov(z_min)
         r_comov_min = max(0., r_comov_min + xcf.r_par_min)
         args.z_min_obj = cosmo.distance_to_redshift(r_comov_min)
-        userprint("\r z_min_obj = {}\r".format(args.z_min_obj), end="")
+        userprint("\r z_min_obj = {}\r".format(args.z_min_obj) )
```
Comment (julianbautista) [inline bin/picca_metal_xdmat.py:290]: Maybe we can remove the `\r` in these calls.

File: bin/picca_xcf.py [hunk 3]
```diff
@@ -297,12 +297,12 @@ def main():
         r_comov_min = cosmo.get_r_comov(z_min)
         r_comov_min = max(0., r_comov_min + xcf.r_par_min)
         args.z_min_obj = cosmo.distance_to_redshift(r_comov_min)
-        userprint("\r z_min_obj = {}\r".format(args.z_min_obj), end="")
+        userprint("\r z_min_obj = {}\r".format(args.z_min_obj) )
```
Comment (julianbautista) [inline bin/picca_xcf.py:300]: same as the other file

File: bin/picca_xdmat.py [hunk 4]
```diff
@@ -257,12 +257,12 @@ def main():
         r_comov_min = cosmo.get_r_comov(z_min)
         r_comov_min = max(0., r_comov_min + xcf.r_par_min)
         args.z_min_obj = cosmo.distance_to_redshift(r_comov_min)
-        userprint("\r z_min_obj = {}\r".format(args.z_min_obj), end="")
+        userprint("\r z_min_obj = {}\r".format(args.z_min_obj) )
```
Comment (julianbautista) [inline bin/picca_xdmat.py:260]: same as the other file

**Review by julianbautista — CHANGES_REQUESTED** [review]
I suggested some small changes. Otherwise seems fine to me.

**Comment by SamYoules** [comment]
Thanks Julian. I've made those changes, run pytest again and have pushed
them to logfiles.
Sam

**Review by julianbautista — APPROVED** [review]
Looks good to me. Thanks Sam

**Pull request merged**